src/taskcreator.js

```diff
@@ -79,7 +93,42 @@ class TaskCreator {
       this.lastCreateTask = {taskId: options.taskId, task};
       return {status: {taskId: options.taskId}};
     }
-    return await queue.createTask(options.taskId, task);
+
+    let lastFire, taskCreateRes;
+    try {
+      taskCreateRes = await queue.createTask(options.taskId, task);
+      lastFire = {
+        result: 'success',
+        taskId: options.taskId,
+        time: new Date(),
+      };
+    } catch (err) {
+      lastFire = {
+        result: 'error',
+        error: err,
+        time: new Date(),
+      };
+    }
+
+    try {
+      await this.appendLastFire({
+        hookGroupId: hook.hookGroupId,
+        taskCreateTime: lastFire.time, 
+        hookId: hook.hookId,
+        firedBy: context.firedBy,
+        taskId: lastFire.taskId,
+        result: lastFire.result,
+        error: lastFire.error || '',
```

Comment:
I think this needs to convert `lastFire.error` into a string (so `lastFire.error.toString()`.  Is there a test for the error case?